private/buf/bufgen/internal/bufgenv2/testdata/managed/match.yaml

```diff
@@ -0,0 +1,46 @@
+version: v2
+plugins:
+  - binary: "protoc-gen-go"
+    out: "gen/out"
+managed:
+  enabled: true
+  disable:
+    - path: excluded/
+    - file_option: java_package
+      path: java-excluded/
+    - file_option: java_package
+      module: buf.build/acme/weather
+    - file_option: java_package
+      path:  a/b/c.proto
+    - file_option: java_package
+      module: buf.build/acme/petapis
+      path: x/y/z.proto
+    - file_option: java_package
+      path: exact.proto
+  override:
+    - option: java_package
```

Comment:
Why is this `option` in `override` but `file_option` in disable? This should be the same thing everywhere. I thought we agreed on `file_option` in the latest iteration, but instructions remain the same: I should not have to think about what is in override vs disable, I shouldn't have to read the docs on this

Reply:
sorry I forgot to change the field tag